services/wallet/api.go

```diff
@@ -223,11 +239,11 @@ func (api *API) AddFavourite(ctx context.Context, favourite Favourite) error {
 	return err
 }
 
-func (api *API) GetCryptoOnRamps(ctx context.Context) ([]CryptoOnRamp, error) {
+func (api *API) GetCryptoOnRamps(ctx context.Context) ([]*CryptoOnRamp, error) {
```

Comment:
I'm curious, what benefit do we get from passing a slice of references over a slice of values?

Reply:
Isn t it better to pass the pointer rather than copy the whole CryptoOnRamp?

Reply:
No, a slice index is a pointer effectively. A slice pointer is much slower.

Reply:
TIL, I will update the PR accordingly, thanks